src/TextEditor/monaco_editor_page.h

```diff
@@ -0,0 +1,15 @@
+#ifndef MONACO_EDITOR_PAGE_H
+#define MONACO_EDITOR_PAGE_H
+
+#include <QWebEnginePage>
+
+class MonacoEditorPage : public QWebEnginePage {
```

Comment:
Please move under FOEDAG namespace

Reply:
moved CPPEndPoint and MonacoEditorPage classes under FOEDAG namespace.